secboot/secboot_tpm.go

```diff
@@ -590,6 +590,9 @@ func SealKeys(keys []SealKeyRequest, params *SealKeysParams) ([]byte, error) {
 			return nil, err
 		}
 
+		if key.SlotName == "default" {
+			key.BootstrappedContainer.RegisterKey(primaryKeyOut, unlockKey)
```

Comment:
primaryKeyOut should be the same as primaryKey, right?

Reply:
Well, `primaryKey` might be `nil`. But if it is not, they should be the same. The correct one is always `primaryKeyOut`.